fatrace.c

```diff
@@ -225,6 +226,86 @@ show_pid (pid_t pid)
     return true;
 }
 
+static bool
+print_json_str (const char* key, const char* value) {
+    const unsigned char* str = (unsigned char*)value;
+    printf("\"%s\":\"", key);
+    bool decode_problem = false;
+    for (int i = 0; str[i] != 0; ) {
+        char c = str[i];
```

Comment:
For the processing below it feels safer and more natural to use an `unsigned char` for this.

Reply:
Fixed in #51